src/lib/io/sol-iio.c

```diff
@@ -157,6 +157,8 @@ craft_filename_path(char *path, size_t size, const char *base, ...)
     len = vsnprintf(path, size, base, args);
     va_end(args);
 
+    SOL_DBG("Crafted path len: %d Buffer size: %zu", len, size);
```

Comment:
On 26-01-2016 16:45, Gustavo Sverzut Barbieri wrote:
> In src/lib/io/sol-iio.c
> <https://github.com/solettaproject/soletta/pull/1321#discussion_r50879574>:
>
>> @@ -157,6 +157,8 @@ craft_filename_path(char *path, size_t size, const char *base, ...)
>>      len = vsnprintf(path, size, base, args);
>>      va_end(args);
>>
>> +    SOL_DBG("Crafted path len: %d Buffer size: %zu", len, size);
>
> the important information should be the path, which is not printed. You
> can shrink it to: "available=%zu, used=%d, path=\"%s\""

Well, if `vsnprintf` failed, path is undefined, no? It's better not try 
to print it... I was willing to avoid ifs and branching just for logging...

>
> You can omit "crafted" since it's going to show the function name.
>
Fair.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/1321/files#r50879574>.
>